apps/assisted_tagging/serializers.py

```diff
@@ -127,3 +128,52 @@ def validate(self, data):
                 association='Association is required for this widget.'
             ))
         return data
+
+
+class TriggeredDraftEntryGqlSerializer(
+        DraftEntryBaseSerializer,
+        ProjectPropertySerializerMixin,
+        UserResourceCreatedMixin, serializers.ModelSerializer):
+    class Meta:
+        model = DraftEntry
+        fields = (
+            'lead',
+        )
+
+    def create(self, data):
+        lead = Lead.objects.get(id=self.data['lead'])
+        if lead.auto_entry_extraction_status == (Lead.AutoExtractionStatus.SUCCESS):
+            raise serializers.DjangoValidationError("Already Tiggered")
+        if not lead.leadpreview.text_extract:
+            raise serializers.DjangoValidationError('Simplifed Text is empty')
+        draft_entry = DraftEntry.objects.filter(lead=self.data['lead'], draft_entry_type=0)
+        if draft_entry.exists():
+            raise serializers.DjangoValidationError('Draft entry already exit')
+        # Use already existing draft entry if found
+        # Create new one and send trigger to deepl
+        lead.auto_entry_extraction_status = Lead.AutoExtractionStatus.PENDING
+        lead.save(update_fields=['auto_entry_extraction_status'])
+        transaction.on_commit(
+            lambda: trigger_request_for_auto_draft_entry_task.delay(self.data['lead'])
```

Comment:
self.data['lead'] should be an object.
```suggestion
            lambda: trigger_request_for_auto_draft_entry_task.delay(lead.id)
```